### ISSUE TITLE
Set up Usage info for 'past agreement date' RUP's on initial Create

### ISSUE THREAD

**Issue by micheal-w-wells** [bcgov/range-api#60]
The need to extend Usage past agreement dates in FTA means the Create RUP will need to be done prior to a Range officer leaving cell service.  Putting the logic in the API will save us from baking the same smarts into both the iOS app and the web app.

**Comment by micheal-w-wells** [comment]
parked this one and did web only as that was a bit faster to get out, and still gets us closer to the right information on the PDF

**Comment by LisaMoore1** [comment]
@micheal-w-wells Does this one need to stay in next sprint or is it open for prioritizing?

**Comment by LisaMoore1** [comment]
@micheal-w-wells Will this issue see any action this sprint or does it belong somewhere else for now?

**Comment by micheal-w-wells** [comment]
@LisaMoore1 can ice this one, we're just rolling with the range-web solution for now

**Comment by calebissharp** [comment]
Closing in favour of clientside implementation.
